packages/go_router/README.md

```diff
@@ -38,6 +38,7 @@ See the API documentation for details on the following topics:
 - [State restoration](https://pub.dev/documentation/go_router/latest/topics/State%20restoration-topic.html)
 
 ## Migration Guides
+- [Migrating to 17.0.0](https://flutter.dev/go/go-router-v17-breaking-changes).
```

Comment:
for the doc content, instead of using word `merge`, we should state its effect instead.

e.g.

instead of
`ShellRoute should merge GoRouter's observers by default.`

we should state this instead

`ShellRoute's navigating changes notify GoRouter's observers by default.`